dp/generate_climos.py

```diff
@@ -134,19 +134,34 @@ def climo_outputs(time_resolution):
     # Apply per-variable processing
     # - Scale pr variable if it is not in desired units
     #   - TODO: Use UDUNITS here for more robust processing of units? It has a fairly heavy API. But this is uuuugly.
```

Comment:
There is another Python binding of udunits2 to Python,[ `cf_units`](https://github.com/SciTools/cf_units). It appears to be Python 3 compatible. It's written by the UK Met office, and appears to be actively maintained. Upside: Exactly what is called for in this situation. Downside: Have to install `udunits2` separately.